recipe/meta.yaml

```diff
@@ -39,6 +40,7 @@ requirements:
     - requests >=2.20.1,<3
     - ruamel_yaml >=0.11.14,<0.16
     - setuptools >=31.0.1
+    - toolz >=0.8.1
```

Comment:
```suggestion
    # conda-build to drop cytoolz dependency (https://github.com/conda/conda-build/pull/4556)
    # once that occurs this can be relaxed to toolz (https://github.com/conda/conda/pull/11700)
    - cytoolz >=0.8.1
```

Reply:
No longer necessary with https://github.com/conda/conda/pull/11704